openmetadata-spec/src/main/resources/json/schema/entity/services/connections/database/sapHana/sapHanaSQLConnection.json

```diff
@@ -0,0 +1,38 @@
+{
+  "$id": "https://open-metadata.org/schema/entity/services/connections/database/sapHana/sapHanaConnection.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "SapHanaSQLConnection",
+  "description": "Sap Hana Database SQL Connection Config",
+  "type": "object",
+  "javaType": "org.openmetadata.schema.services.connections.database.sapHana.SapHanaSQLConnection",
```

Comment:
These classes were before a definition. However, I could not get a java object generated out of that. Shape is the same, just moved to their own file